docs/book/features/file_volume.md

```diff
@@ -11,6 +11,8 @@ Proceed to the requirements section below to enable this feature in your environ
 Check the [CSI vSphere compatibility matrix](../compatiblity_matrix.md) and the [supported features](../supported_features_matrix.md) section to verify if your environment conforms to all the required versions. Also check the [limits](../limits.md) section to understand if this feature can cater to your needs.
 In order to utilize this feature in your vSphere environment, you need to make sure of the following:
 
+- Verify if your vSphere user for CSI driver has the required roles and privileges to create file volumes. Refer to the roles and privileges section in the [CSI driver deployment pre-requisites](../driver-deployment/prerequisites.md) page for more information.
```

Comment:
@divyenpatel Do I have to mention here that the customer might have to restart vpxd after this step?

Reply:
`Verify if your vSphere user for CSI driver...` => `Verify vSphere user for CSI driver...`

Reply:
No, we do not need to mention vpxd restart. we are looking to issue and the fix.